helix/transforms/tigress/tigress.py

```diff
@@ -0,0 +1,183 @@
+import copy
+import logging
+import os
+import shutil
+from urllib import parse
+from urllib import request as req
+
+import magic
+
+from ... import exceptions, transform, utils
+from . import utils as tigress_utils
+from .configurations import tigress_transforms
+
+logger = logging.getLogger("transform.tigress")
```

Comment:
`logging.getLogger(__name__)` is a more standard approach and would work well with https://github.com/helix-datasets/helix/issues/26